torch/_inductor/kernel/flex_attention.py

```diff
@@ -425,10 +465,28 @@ def flex_attention(*args, **kwargs):
     # Note, we don't need to pass in the captured buffers explicitly
     # because they're implicitly added by the score_mod function
     # We do need to explicitly pass it in for autotuning though.
+
+    q_len = query.get_size()[-2]
+    kv_len = key.get_size()[-2]
+    sparse_mask_q_blocks = sparse_mask_kv_indices.get_size()[0]
+    sparse_mask_kv_blocks = sparse_mask_q_indices.get_size()[0]
+    BLOCKSPARSE_Q = q_len // sparse_mask_q_blocks
+    BLOCKSPARSE_KV = kv_len // sparse_mask_kv_blocks
```

Comment:
TODO: This doesn't work well with dynamic shape, since either q_len or kv_len can be symbolic. ```BLOCKSPARSE_Q``` and ```BLOCKSPARSE_KV``` should be a configuration. This is just for backward test compatibility.